mne_connectivity/tests/test_var.py

```diff
@@ -0,0 +1,157 @@
+import numpy as np
+import pytest
+from numpy.testing import (
+    assert_array_almost_equal, assert_array_equal,
+    assert_almost_equal
+)
+
+from mne_connectivity import vector_auto_regression
+
+
+np.random.seed(12345)
```

Comment:
do not set global seeds if you can. It can have side effects far in the code

Reply:
Used RandomState.